viam-cartographer.go

```diff
@@ -470,7 +477,9 @@ func (slamSvc *builtIn) GetInternalStateStream(ctx context.Context, name string)
 }
 
 // New returns a new slam service for the given robot.
-func New(ctx context.Context, deps registry.Dependencies, config config.Service, logger golog.Logger, bufferSLAMProcessLogs bool) (slam.Service, error) {
+func New(ctx context.Context, deps registry.Dependencies, config config.Service,
+	logger golog.Logger, bufferSLAMProcessLogs bool,
```

Comment:
To confirm, this change is to obey the line width limit?

Curious why there needs to be a comma after `bufferSLAMProcessLogs bool`?

Reply:
That's correct. I looked up why a comma is needed: without the comma, the lexer will insert a semicolon after `bool` which is invalid: https://go.dev/doc/effective_go#semicolons

Reply:
Got it. It looked a bit unusual because there two arguments on that line, and I think typically a trailing comma is only used if each argument is on its own line ([example](https://github.com/viamrobotics/viam-cartographer/blob/fe8feabb09e429c6e0d89378aae1557740dabcb4/viam-cartographer.go#L619)).

I couldn't find Go documentation saying that, but for python: https://peps.python.org/pep-0008/#when-to-use-trailing-commas.

Would you mind pinging one of the Golang experts (Tess showed me [this wiki](https://viam.atlassian.net/wiki/spaces/ENG/pages/9732097/Language+Framework+Reviewers) yesterday) to confirm this is okay, or if it's something we'd want to change in the linter?

If you can't get an answer quickly, feel free to push this PR; I made a note for myself to check in about it.

Reply:
Sure! @edaniels what are your thoughts on the above?

Reply:
Cool, that makes sense to me. It'd be nice if the linter did/suggested this automatically - is there a place to file a ticket for that?

Reply:
Oh, ok haha. Sounds good!

Reply:
you'd need to file that against golangci-lint probably and whatever linter complained. it's probably not worth it imo

Reply:
I see, got it thanks!